torch/_inductor/codegen/common.py

```diff
@@ -777,8 +797,34 @@ class CSEProxy:
             @staticmethod
             def __getattr__(name):
                 def inner(*args, **kwargs):
+                    buf_bounds = ValueRanges.unknown()
+                    if name == "constant":
```

Comment:
Note to self: This shouldnt' be necessary. Not sure why was it not being found in the bounds. I'll look into it.